client/src/message.rs

```diff
@@ -13,24 +13,24 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-//! Defines [Call] trait and implementations for all transaction parameters.
+//! Defines [Message] trait and implementations for all transaction parameters.
```

Comment:
We can be a bit more precise here:
```suggestion
//! Defines [Message] trait and implementations for all messages in `radicle_registry_core::messages`.
```